apps/api-gateway/src/public-data/public-data.controller.ts

```diff
@@ -0,0 +1,29 @@
+import { Controller, Get, Param } from '@nestjs/common';
+import { ApiCreatedResponse, ApiOperation, ApiTags } from '@nestjs/swagger';
+import { PublicDataService } from './public-data.service';
+
+@Controller('pub-data')
```

Comment:
이렇게 작성해 주는 건 어떨까요?

약어를 사용하는 건 취향의 영역이라고 생각하지만 개인적으로는 많이 사용하는 약어가 아닌 이상은 임의로 줄이는건 좋지 않다고 생각하고(저는 많이 쓰는 약어도 풀네임으로 쓰는 걸 좋아하긴 해요) 이 경우에는 pub이라는 용어가 publish와 헷갈릴 수도 있을 것 같아요.
```suggestion
@Controller('public-data')
```

Reply:
확실히 그럴 것 같습니다
해당 부분도 수정후 커밋으로 반영후 추가 코멘트
올리겠습니다!

Reply:
해당 부분 수정하여 
[62ce98e](https://github.com/GrebbMe/grabbme-be/pull/13/commits/62ce98ec4578624c249a6876f0caa8adce60a020)
에 반영하였습니다!

- public-data EndPont 변경 : pub-data -> public-data